chrome-extension/sidebar/sidebar.js

```diff
@@ -48,6 +48,307 @@ const htmlToElement = (html) => {
   return template.content.firstChild;
 };
 
+// Begin code for sorting course list
+document.addEventListener('change', function(event) {
+  if (event.target.matches('.size-dropdown')) {
+    handleDropdownChange(event);
+  }
+});
+
+// padding: 6px 10px;
+// padding: 4px 10px;
+const dropdownTemplate = `
+<div style="display: inline-block; margin-left: -20px; margin-right: 5px;">
+  <select class="size-dropdown" style="height: 34px; transform: translateY(3%); font-size: 14px; padding: 6px 12px;">
+    <option value="A">Sort By Course # (Default)</option>
```

Comment:
Let's use more meaningful values here instead of ABCDE, makes the code below harder to read and check for correctness if we use unrelated single letters instead of values like "sortCourseNumASC" or similar